scrapers/il/bills.py

```diff
@@ -321,23 +324,17 @@ def scrape(self, session=None):
                     chamber, session_id, "AM", bill_url, "appointment"
                 )
 
-            # TODO: get joint session resolution added to python-opencivicdata
-            # for bill_url in self.get_bill_urls(chamber, session_id, 'JSR'):
-            #     bill, votes = self.scrape_bill(chamber, session_id, 'JSR', bill_url,
-            #                                    'joint session resolution')
-            #     yield bill
-            #     yield from votes
-
     def scrape_archive_bills(self, session):
         session_abr = session[0:2]
-        url = f"https://www.ilga.gov/legislation/legisnet{session_abr}/{session_abr}gatoc.html"
+        url = f"https://beta.ilga.gov/documents/legislation/legisnet{session_abr}/{session_abr}gatoc.html"
```

Comment:
Please use a `BASE_URL` constant (or similar variable) in place of the literal string `https://beta.ilga.gov`

Reply:
Fixed issue